argocd/applications/custom/infra-onboarding.tpl

```diff
@@ -40,6 +40,10 @@ infra-config:
     orchAttestationStatus: attest-node.{{ .Values.argo.clusterDomain }}:443
     orchRegistry: {{ .Values.argo.releaseService.ociRegistry }}:9443
     orchFileServer: {{ .Values.argo.releaseService.fileServer }}:60444
+    orchMPSHost: mps-node.{{ .Values.argo.clusterDomain }}:4433
+    orchMPSWHost: mps-webport-node.{{ .Values.argo.clusterDomain }}:443
+    orchRPSHost: rps-node.{{ .Values.argo.clusterDomain }}:443
+    orchRPSWHost: rps-webport-node.{{ .Values.argo.clusterDomain }}:443
```

Comment:
add a comment that these routes need to be revisited - not all of them need to be exposed.

Reply:
we should start using the MT GW endpoints